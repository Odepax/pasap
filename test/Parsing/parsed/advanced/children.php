<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<news>
			<about>
				<author>Me</author>
				<created>12-12-12</created>
			</about>
			<content>
				<title>We are Amazing</title>
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque, consequatur cum iste laudantium libero magni maxime
					minima mollitia quos. At earum nostrum pariatur. Amet, at
					fugiat incidunt officia possimus vel?
				</p>
			</content>
		</news>
	</body>
</html>
