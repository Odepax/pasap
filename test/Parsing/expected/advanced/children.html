<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<article class="News">
			<h1 class="title">We are Amazing</h1>
			<div class="about">
				<em>
					By Me,
					At 12-12-12
				</em>
			</div>
			<div class="content">
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque, consequatur cum iste laudantium libero magni maxime
					minima mollitia quos. At earum nostrum pariatur. Amet, at
					fugiat incidunt officia possimus vel?
				</p>
			</div>
		</article>
	</body>
</html>
